src/lint/linter/ArcanistESLintLinter.php

```diff
@@ -15,7 +15,7 @@ public function getInfoURI() {
     }
 
     public function getRuleDocumentationURI($ruleId) {
-        return $this->getInfoURI().'/docs/rules/'.$ruleId;
+      return $this->getInfoURI().'/docs/rules/'.$ruleId;
```

Comment:
seems like the rest of this file uses 4-space indent -- can we keep it consistent here please?